docs/src/pages/components/buttons/buttons.md

```diff
@@ -107,7 +107,7 @@ Here is an [integration example with react-router](/guides/composition/#button).
 
 ### Cursor not-allowed
 
-The ButtonBase component sets `pointer-events: none;` on disabled buttons, which prevents the appearance of a disabled cursor.
+The `ButtonBase` component sets `pointer-events: none;` on disabled buttons, which prevents the appearance of a disabled cursor.
```

Comment:
I believe that we didn't settle on a formating strategy for the components yet. There is a concern with the style of \`\`.

```suggestion
The ButtonBase component sets `pointer-events: none;` on disabled buttons, which prevents the appearance of a disabled cursor.
```

Reply:
Hey @oliviertassinari not sure if i'm following you on this one. I noticed all the ButtonBase are decorated as code, and it felt like a bit inconsistency here. That's the reason behind this change. Let me if there's any concern or special case addressed for this one, i can revert it back :) Thx

Reply:
I'm glad you raise this consitency topic, I believe that if we take wall the pages into account, it's highly inconsistent (50/50 usage split between these two forms). I wonder about what would be the best strategy.

Reply:
cc @mbrookes.

Reply:
It used to be that the name of the UI pattern (e.g. 'expansion panel') was not formatted, but the name of the component `ExpansionPanel` was. I have no string preference whether it is or isn't formatted, once it's consistent.

Reply:
@mbrookes sounds great